pkgs/dart_services/lib/src/flutter_genui.dart

```diff
@@ -89,12 +89,11 @@ class GenUi {
   }
 
   Future<String> generateCode({required String prompt}) async {
-    final result = await _prod.request(prompt: prompt);
-
-    if (result == null) {
-      throw Exception('Failed to generate code from GenUI');
+    try {
+      return await _prod.request(prompt: prompt);
+    } catch (e) {
+      _logger.warning('Genui failed: $e');
```

Comment:
There's a note above about logger only supporting one-line statements. Is Exception.toString guaranteed to be an oneliner?

Reply:
thanks

added formatting